### PR TITLE
add package data for VERSION file to work

### PR DESCRIPTION
**Pull request by nmartin413**
need to explicitly include the `VERSION` file ++

### CODE REVIEW

**Comment by maxrichie5** [comment]
What does this do??

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1 @@
+include VERSION
```
Comment (maxrichie5) [inline MANIFEST.in:1]: Why isn't the README needed here?

**Comment by nmartin413** [comment]
@maxrichie5 - it seems like there's a special case where the README is automatically included as package data. This change explicitly also includes the `VERSION` file in the package data.

File: MANIFEST.in (hunk 1 above)
Comment (nmartin413) [reply MANIFEST.in:1]: Seems like that's a special case.

**Review by maxrichie5 — APPROVED** [review]

**Pull request merged**